maps/map_files/Ice_Colony_v3/sprinkles/40.shivabarls.dmm

```diff
@@ -0,0 +1,1959 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"ak" = (
```

Comment:
- Generate Icons by Icon State and place those etc.
- ![bilde](https://user-images.githubusercontent.com/59782240/184453102-982f01b9-9bab-41c2-919d-44566b55efaf.png)
- A lot of object clutter.
- CLF equipment somehow better equipped than the UPP on the bar with multiple medkits, weapons and ammo. 
- The colored parkas are cool.
- Damaged cades and health, you know what to do.
- Noop turfs/area as previously said in above comments.
- The dog tag bottom left has no information, give it some information for that sweet lore.